assets/src/edit-story/app/story/effects/test/useLoadStory.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { mount } from 'enzyme';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import useLoadStory from '../useLoadStory';
+import { createStory } from '../../../api/testUtils/story';
+import { useAPI, useHistory } from '../../../';
+
+jest.mock('../../../', () => ({
+  useHistory: jest.fn(),
+  useAPI: jest.fn(),
+}));
```

Comment:
To mock these, we normally create a valid context provider as a wrapper for the component, with mock functions. In this case, you don't even need the response from `clearHistory`, so it doesn't need to be a mock function, just a dummy function.

See for instance [`library/test/_utils/index.js`](https://github.com/google/web-stories-wp/blob/master/assets/src/edit-story/components/library/test/_utils/index.js).

Reply:
Done.

Reply:
I've also mocked clearHistory, since it calls a reducer dispatch which seems to prevent the  getStoryWithId await from finishing.